### ISSUE TITLE
"Unable to parse environment variable as key/value pair" error during release

### ISSUE THREAD

**Issue by alilleybrinker** [axodotdev/cargo-dist#1407]
See: https://github.com/mitre/hipcheck/actions/runs/10782669325/job/29903109292

This appears to be specific to the two macOS targets (`aarch64-apple-darwin` and `x86_64-apple-darwin`). Builds for Windows and Ubuntu appear to have made it past this initial stage just fine.

**Comment by ashleygwilliams** [comment]
oh yikes. let me take a look, sorry you ran into this.

**Comment by alilleybrinker** [comment]
Looks like `cargo-dist` is failing to split the `HOMEBREW_HELP_MESSAGE` environment variable run it runs `brew bundle exec -- /usr/bin/env`. When I run that command myself locally, this is what that environment variable looks like:

```
HOMEBREW_HELP_MESSAGE=Example usage:
  brew search TEXT|/REGEX/
  brew info [FORMULA|CASK...]
  brew install FORMULA|CASK...
  brew update
  brew upgrade [FORMULA|CASK...]
  brew uninstall FORMULA|CASK...
  brew list [FORMULA|CASK...]

Troubleshooting:
  brew config
  brew doctor
  brew install --verbose --debug FORMULA|CASK

Contributing:
  brew create URL [--no-fetch]
  brew edit [FORMULA|CASK...]

Further help:
  brew commands
  brew help [COMMAND]
  man brew
  https://docs.brew.sh
```

**Comment by ashleygwilliams** [comment]
error thrown: https://github.com/axodotdev/cargo-dist/blob/6e55304478309a20bb3b9ac959f2f079f6e60a20/cargo-dist/src/errors.rs#L309-L316

from: https://github.com/axodotdev/cargo-dist/blob/6e55304478309a20bb3b9ac959f2f079f6e60a20/cargo-dist/src/build/cargo.rs#L148

**Comment by ashleygwilliams** [comment]
betting the culprit is in this https://github.com/axodotdev/cargo-dist/commit/0344954e59c85967039eb84e3aa94b52862bdaca

**Comment by Gankra** [comment]
https://github.com/axodotdev/cargo-dist/commit/0344954e59c85967039eb84e3aa94b52862bdaca is in fact the fix for this issue (which was introduced by changes brew itself made last month).

The fix shipped in 0.21.1, but the above release was cut with 0.21.0, so it didn't have the fix. So I believe "just update cargo-dist" is ultimately the resolution.

**Comment by alilleybrinker** [comment]
Okay, great! We did a workaround to use a custom task to get our immediate release out. Tomorrow we can undo the workaround and just use the latest without it.

Thanks so much for the quick help!
